templates/registration/password_reset_confirm.html

```diff
@@ -49,7 +49,7 @@ <h5>Password Guidance</h5>
         
     {% else %}
         <h1>Password reset failed</h1>
-        <p>The password reset link was invalid, possibly because it has already been used. Please request a new password reset.</p>
+        <p>The password reset link was invalid, possibly because it has already been used or it has expired - tokens only last 72 hours. Please request a new password reset.</p>
```

Comment:
I wouldn't say token here, it doesn't mean anything to a user. Maybe just:

```suggestion
        <p>The password reset link was invalid, possibly because it has already been used or it has expired after 72 hours. Please request a new password reset.</p>
```